app/src/main/assets/input_data/Comments.csv

```diff
@@ -6,3 +6,8 @@
 5,True,6,Poor human player (amp)
 6,True,4,Robot had note(s) stuck in it
 7,False,,They were mean
+8,True,7,Evaded defense well
+9,True,8,Ineffective defense
+10,True,9,Committed a G-424 foul
+11,True,10,Took more than 5 seconds to pick up a note
```

Comment:
"5 seconds" is pretty hard to judge.  Can we say "Clearly struggled to pick up a note" or just "Struggled to pick up a note" ??  I'm okay either way, i just think people mike NOT pick it 'cause it "felt" like 4 seconds.  Just a suggestions.